opencensus/stats/view_descriptor.h

```diff
@@ -26,37 +26,59 @@
 namespace opencensus {
 namespace stats {
 
-// ViewDescriptor provides metadata for a view, including its identity and the
-// data to be collected.
+// ViewDescriptor provides metadata for a view: a unique name, the measure to
+// collect data for, how to aggregate that data, and what tag keys to break it
+// down by.
+// In order to collect data for a ViewDescriptor, it must either be registered
+// for export (by calling RegisterForExport() on the fully-defined descriptor)
+// or converted into a View to collect data on-task (see view.h).
+//
 // ViewDescriptor is a value type, and is thread-compatible.
-// TODO: DOCS: Document members.
 class ViewDescriptor final {
  public:
   //////////////////////////////////////////////////////////////////////////////
   // View definition
 
+  // Creates a ViewDescriptor with Cumulative aggregation.
   ViewDescriptor();
 
+  // Sets the name of the ViewDescriptor. Names must be unique within the
+  // library; it is recommended that it be in the format "<domain>/<path>",
+  // where "<path>" uniquely specifies the measure, aggregation, and columns
+  // (e.g. "example.com/Foo/FooUsage-sum-key1-key2").
   ViewDescriptor& set_name(absl::string_view name);
   const std::string& name() const { return name_; }
 
   // Sets the measure. If no measure is registered under 'name' any View created
   // with the descriptor will be invalid.
   ViewDescriptor& set_measure(absl::string_view name);
-
+  // Accesses the descriptor of the descriptor's measure. If no measure has been
```

Comment:
Descriptor of the measure?

Reply:
Yes. Rewritten to "descriptor of the view's measure".